internal/core/thirdparty/knowhere/CMakeLists.txt

```diff
@@ -11,7 +11,14 @@
 # or implied. See the License for the specific language governing permissions and limitations under the License.
 #-------------------------------------------------------------------------------
 
-set( KNOWHERE_VERSION 9aa3e21 )
+set( KNOWHERE_VERSION add_version_support )
+set( GIT_REPOSITORY  "https://github.com/foxspy/knowhere.git")
```

Comment:
Please utilize the official repository instead and refrain from linking personal repositories.

Reply:
knowhere merged and fixed it.